config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   resources :compliments, only: [:show]
+  get ':slug' => 'organizations#show', as: 'organization'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.